packages/lib/src/components/ExpiringAnnouncementTooltip/index.tsx

```diff
@@ -29,7 +29,7 @@ export const ExpiringAnnouncementTooltip: React.FC<
     setOpen(false);
 
     if (neverSeenTooltip) {
-      Cookies.set(cookieName, seenTooltipDate.toString());
+      Cookies.set(cookieName, seenTooltipDate.toString(), { expires: 365 });
     }
```

Comment:
Question: does `differenceInDays(now, seenTooltipDate) < activeDays)` mean the user is going to see it for 8 days in a row after this? I wonder if it's better to decrease that number for this tooltip now?